rules/hardcoded_credentials.go

```diff
@@ -119,6 +151,14 @@ func (r *credentials) matchEqualityCheck(binaryExpr *ast.BinaryExpr, ctx *gosec.
 				}
 			}
 		}
+		if okLit { // Match for literals
+			s, err := gosec.GetString(identStrConst)
```

Comment:
`err` is unhandled. If it shouldn't propagate the error up, it should probably be logged

Reply:
I am gonna go ahead and resolve this comment, but if there are any questions are anything wrong with my solution, you can reopen it on the latest commit.